src/Apps/serverCacheTTLs.tsx

```diff
@@ -8,6 +8,8 @@ const HOURS_24 = 86400 // In seconds
  */
 export const serverCacheTTLs = {
   artist: HOURS_24,
+  article: NO_CACHE,
+  articles: NO_CACHE,
```

Comment:
It might be nice to retain some cacheability, just to insulate back-ends a little from popular ("viral" 😉) content and improve performance by responding directly from the CDN when possible. A TTL like 5 or 10 minutes might provide that without affecting business operations much.

Reply:
That's a good idea - easy enough and unlikely to be 'noticed' in terms of 'woah caching is busted - its taking forever to update'.

The positron data loaders in MP already take advantage of data loader caching (leaving off access token) - which is 10 minutes in production already.

So I think I'll change this one to 5 minutes.